### PR TITLE
Emergency contacts

### LINKED ISSUE #133: Add "emergency contact" field
Currently, there's a "phone parents" field for members. To reflect changes in the CH registration form, we're going to need to change the name of this field to "In Case of Emergence (ICE) phone number" and add another "ICE Contact Name" field.

### PR DESCRIPTION
**Pull request by svenvanhal**
Fixes #133

### CODE REVIEW

**Comment by svenvanhal** [comment]
Todo: translation

**Comment by praseodym** [comment]
* https://github.com/WISVCH/dienst2-api-client-java/blob/master/src/main/java/ch/wisv/dienst2/apiclient/model/Student.java moet ook worden aangepast

File: ldb/templates/ldb/person_detail.html [hunk 1]
```diff
@@ -87,8 +87,14 @@ <h2 class="page-header">
...
-          {% if person.student %}
-            {% stcontrol_format person.student 'phone_parents' '<a href="tel:{0}"><i class="fa fa-fw fa-phone"></i>{0}</a>' %}
+          {% if person.student.emergency_phone %}
+            <div class="form-group">
+              <label class="col-sm-4 control-label">Contact noodgeval</label>
+              <div class="col-sm-8">
+                <p class="form-control-static"><a href="tel:{{ person.student.emergency_phone }}"><i class="fa fa-fw fa-phone"></i>{{ person.student.emergency_phone }}</a>
+                  {% if person.student.emergency_name %}({{ person.student.emergency_name }}){% endif %}</p>
+              </div>
+            </div>
```
Comment (jgadelange) [inline ldb/templates/ldb/person_detail.html]: Wouldn't it be better if we just added two `{% stcontrol_format [...] %}`s? (if you don't agree you can just merge)

Comment (praseodym) [reply ldb/templates/ldb/person_detail.html]: Agreed!

Comment (svenvanhal) [reply ldb/templates/ldb/person_detail.html]: Doesn't really make sense to display the emergency contact when there's no phone number available, but also doesn't hurt to just show all information we have.

File: ldb/models.py [hunk 2]
```diff
@@ -410,7 +410,8 @@ class Meta:
     first_year = models.IntegerField(_('first year'), blank=True, null=True)
     student_number = CharNullField(_('student number'), max_length=7, blank=True, null=True, unique=True)
 
-    phone_parents = models.CharField(_('phone parents'), max_length=16, blank=True)
+    emergency_name = models.CharField(_('emergency name'), max_length=16, blank=True)
+    emergency_phone = models.CharField(_('emergency phone'), max_length=16, blank=True)
```
Comment (Fastjur) [inline ldb/models.py]: Some phone numbers may be longer than 16 chars, right? With international students having country codes in their phone numbers etc. Also, a max length of 16 for a name, shouldn't this be more?

Comment (praseodym) [reply ldb/models.py]: No, the maximum is 15 digits: https://en.wikipedia.org/wiki/MSISDN

Comment (Fastjur) [reply ldb/models.py]: Alright. I will update the emergency name to something bigger then. Say 48 chars.

File: ldb/templates/ldb/person_detail.html [hunk 3]
```diff
@@ -87,9 +87,8 @@ <h2 class="page-header">
           {% stcontrol_format person 'email' '<a href="mailto:{0}">{0}</a>' %}
           {% stcontrol_format person 'phone_fixed' '<a href="tel:{0}"><i class="fa fa-fw fa-phone"></i>{0}</a>' %}
           {% stcontrol_format person 'phone_mobile' '<a href="tel:{0}"><i class="fa fa-fw fa-mobile"></i>{0}</a>' %}
-          {% if person.student %}
-            {% stcontrol_format person.student 'phone_parents' '<a href="tel:{0}"><i class="fa fa-fw fa-phone"></i>{0}</a>' %}
-          {% endif %}
+          {% stcontrol person.student 'emergency_name' %}
```
Comment (jgadelange) [inline ldb/templates/ldb/person_detail.html]: The `if person.student` is removed. Does that break something or was it unnecessary before?

Comment (Fastjur) [reply ldb/templates/ldb/person_detail.html]: Good catch. This caused errors when no emergency_name was given. I've moved it back to inside an `if person.student` block.

**Pull request merged**